v2/azure-arm.yaml

```diff
@@ -683,6 +683,48 @@ status:
 #     sub-resource.
 #
 objectModelConfiguration:
+  apimanagement:
+    2022-08-01:
+      # We dont want to create APIM from this operator, it takes over an hour to create
+      # Assume that this operator configures resource in an existing APIM instance (service)
+      # Service:
+      #   $export: true
+      #   $supportedFrom: v2.4.0
+      # # Link in the Service resource to the other resources
+      # RemotePrivateEndpointConnectionWrapper:
+      #   Id:
+      #     $armReference: true
+      Service_Backend:
```

Comment:
minor: Might be worth using the `exportAs` directive on these to remove the `_` in the exported resource name

Reply:
Love it! Done!